src/main/java/coupon/domain/Coupon.java

```diff
@@ -0,0 +1,70 @@
+package coupon.domain;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+import jakarta.persistence.AttributeOverride;
+import jakarta.persistence.AttributeOverrides;
+import jakarta.persistence.Column;
+import jakarta.persistence.Embedded;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor
+@Getter
+public class Coupon {
+    private static final int MIN_PERCENT = 3;
+    private static final int MAX_PERCENT = 20;
```

Comment:
```suggestion
@Getter
public class Coupon {

    private static final int MIN_PERCENT = 3;
    private static final int MAX_PERCENT = 20;
```

클래스 개행 기준이 일관적이면 좋겠네요!